### PR TITLE
Added start and end time data to context['game']

### PR DESCRIPTION
**Pull request by hiroyuki-komatsu**
* start_time: start time of the game in second since the epoch time
* end_time: end time of the game in second since the epoch time
* start_offset_msec: value of context['game']['msec'] on stat_time
* end_offset_msec: value of context['game']['msec'] on end_time

### CODE REVIEW

**Comment by hiroyuki-komatsu** [comment]
The following change is the next PR after this PR.

https://github.com/hasegaw/IkaLog/commit/76dd1272653ff6580afcee26eb48f73c6fb539e1

File: ikalog/engine.py [hunk 1]
```diff
@@ -194,12 +203,25 @@ def create_context(self):
...
         self.call_plugins('on_game_session_end')
         self.reset()
 
     def session_abort(self):
         self.session_close_wdt = None
 
+        if not self.context['game']['end_time']:
+            # end_time should be initialized in GameFinish or session_close.
+            # This is a fallback in case they were skipped.
+            self.context['game']['end_time'] = IkaUtils.getTime(context)
```
Comment (eagletmt) [inline ikalog/engine.py]: `context` is undefined here. Should be `self.context` ?

```
% python IkaLog.py
IkaLog Primary CLI Language: en_US.UTF-8 (set LANG to override)
IkaLog Game Language: ja (set IKALOG_LANG to override)
<ikalog.utils.icon_recoginizer.weapon.WeaponRecoginizer object at 0x7f47165c7080>: KNN Trained (8118 samples)
<ikalog.utils.icon_recoginizer.gearpower.GearPowerRecoginizer object at 0x7f47165c70f0>: KNN Trained (1010 samples)
<ikalog.inputs.filters.warp_model.WarpFilterModel object at 0x7f4712c59e48>: Loaded model data
  /home/eagletmt/.ghq/github.com/hasegaw/IkaLog/data/webcam_calibration.ja.model (3010 keypoints)
(ERROR)icvOpenAVI_XINE(): Unable to open source 'videotestsrc ! videoconvert ! appsink'
Traceback (most recent call last):
  File "/home/eagletmt/.ghq/github.com/hasegaw/IkaLog/ikalog/engine.py", line 307, in _main_loop
    self.process_frame()
  File "/home/eagletmt/.ghq/github.com/hasegaw/IkaLog/ikalog/engine.py", line 256, in process_frame
    frame, t = self.read_next_frame()
  File "/home/eagletmt/.ghq/github.com/hasegaw/IkaLog/ikalog/engine.py", line 129, in read_next_frame
    frame = self.capture.read_frame()
  File "/home/eagletmt/.ghq/github.com/hasegaw/IkaLog/ikalog/inputs/input.py", line 188, in read_frame
    img = self._read_frame_func()
  File "/home/eagletmt/.ghq/github.com/hasegaw/IkaLog/ikalog/inputs/opencv_file.py", line 88, in _read_frame_func
    raise EOFError()
EOFError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "IkaLog.py", line 101, in <module>
    engine.run()
  File "/home/eagletmt/.ghq/github.com/hasegaw/IkaLog/ikalog/engine.py", line 323, in run
    self._main_loop()
  File "/home/eagletmt/.ghq/github.com/hasegaw/IkaLog/ikalog/engine.py", line 315, in _main_loop
    self.session_abort()
  File "/home/eagletmt/.ghq/github.com/hasegaw/IkaLog/ikalog/engine.py", line 222, in session_abort
    self.context['game']['end_time'] = IkaUtils.getTime(context)
NameError: name 'context' is not defined
```

**Comment by hasegaw** [comment]
you're right

**Comment by hiroyuki-komatsu** [comment]
Ah, sorry and thank you for the fix!

**Pull request merged**